server/RemoteHubClient.ts

```diff
@@ -14,6 +21,10 @@ interface ProjectDetails {
   icon: Buffer;
 }
 interface AddProjectCallback { (err: string, projectId?: string): any; };
+interface ImportProjectData {
+  name: string;
+  data: string;
```

Comment:
I think this should be a `Buffer` not a string? Since it's an ArrayBuffer on the client-side, should be converted to a NodeJS Bon the server-side.